scripts/add_electricity.py

```diff
@@ -131,23 +130,14 @@ def _add_missing_carriers_from_costs(n, costs, carriers):
 def load_costs(tech_costs, config, elec_config, Nyears=1.):
 
     # set all asset costs and other parameters
-    costs = pd.read_csv(tech_costs, index_col=list(range(3))).sort_index()
+    costs = pd.read_csv(tech_costs, index_col=[0,1]).sort_index()
```

Comment:
Using retrieve_cost_data: false breaks here. Problem was that `data/costs.csv` has 'year' as second columns
`resource/costs.csv` doesn't have the 'year' columns.

I tested and would recommend: `index_col=["technology", "parameter"]`